src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +13,14 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
     public boolean login(String email, String password) {
+        User[] users = userService.getUsers();
+        for (User user : users) {
+            if (user.getEmail().equals(email) && user.getPassword().equals(password)) {
```

Comment:
![image](https://github.com/mate-academy/jv-oop-lesson-0/assets/49767049/64ea858f-ec21-4972-84eb-e994d6ee05db)
you already have this logic elsewhere :) Try to reuse it